src/PointwiseFunctions/MathFunctions/RegisterDerivedWithCharm.cpp

```diff
@@ -0,0 +1,15 @@
+// Distributed under the MIT License.
```

Comment:
try to replace with bitbucket stuff from Event and Triggers pull request

Reply:
make sure remaining code is there

Reply:
MathFunctions had the PUP infrastructure added previously already. Adding tests for it now, though